papaya-web/src/main/java/pl/edu/agh/papaya/mappers/Mapper.java

```diff
@@ -0,0 +1,15 @@
+package pl.edu.agh.papaya.mappers;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public abstract class Mapper<M, A> {
+
+    public List<A> mapToApi(List<M> modelItems) {
+        return modelItems.stream()
+                .map(this::mapToApi)
+                .collect(Collectors.toList());
+    }
+
+    public abstract A mapToApi(M item);
+}
```

Comment:
fajne
Czy jest jakiś powód, dla którego nie ma tu konwersji w drugą stronę, którego nie dostrzegam? xd

Reply:
JEST

Reply:
To jest smutne, że o to pytasz, bo była CAŁA DYSKUSJA przy PR Kamila

Reply:
W drugą stronę robimy WIZARDA

Reply:
W drugą stronę dałam tylko przy enumie, bo tam nie ma encji

Reply:
a sorki, tamtego jeszcze nie nadrobiłem

Reply:
ZA PÓŹNO, JEST ZMERGE'OWANE

Reply:
no i co, poczytałem 
nie mam nic do tego wizadra, bo faktycznie ma to sens, że czasami potrzebne są obiekty już zapisane w bazie (w sensie z nadanym id), ale czy przy takim update przychodzącym z frontu (zakładam, że id się nie zmienia przy mapowaniu), nie byłoby łatwiej mieć  taki głupi konwerter i zrobić potem na wszystkim saveOrUpdate()?